doc/api/crypto.markdown

```diff
@@ -727,6 +742,15 @@ treated as the key with no passphrase and will use `RSA_PKCS1_OAEP_PADDING`.
 
 NOTE: All paddings are defined in `constants` module.
 
+## crypto.timingSafeEqual(a, b)
+
+Returns true if a is equal to b, without leaking timing information that would
+help an attacker guess one of the values. This is suitable for comparing secret
+values other than HMACs, such as authentication cookies or
```

Comment:
Why "other than HMACs?" Isn't that exactly what `validate()` is doing, comparing HMACs?